plugins/woocommerce/tests/e2e-pw/tests/merchant/product-import-csv.spec.js

```diff
@@ -92,7 +92,7 @@ const productAttributes = [ 'Color', 'Size' ];
 const errorMessage =
 	'Invalid file type. The importer supports CSV and TXT file formats.';
 
-test.describe( 'Import Products from a CSV file', () => {
+test.describe.serial( 'Import Products from a CSV file', () => {
```

Comment:
Setting this to `serial` mode because the tests here depend on previous ones. Same reasoning noted [here](https://github.com/woocommerce/woocommerce/pull/35492/files#r1013856482).